scripts/ci_test_spelling.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Check arguments when running locally. If running on CI, install dependencies.
+if [ -z "$TRAVIS_PULL_REQUEST" ]; then
+    if [ $# -ne 1 ]; then
+        echo "Usage: ${0##*/} sdk_root_directory"
+        exit 1
+    fi
+
+    # Change to SDK root directory.
+    cd $1
+else
+    set -ev
+    sudo apt-get -y install util-linux    # for gnu getopt
+    sudo apt-get -y install spell         # for spell
```

Comment:
Installs should be moved to `scripts/setup/ci_setup_linux.sh`

Reply:
Done